CRM/Batch/Form/Entry.php

```diff
@@ -637,24 +637,13 @@ private function processContribution(&$params) {
    * @return bool
    */
   private function processMembership(&$params) {
-    $dateTypes = array(
```

Comment:
moved down to the single membership section for clarity, since this does not relate to renewal & causes confusion